src/bun.js/test/jest.zig

```diff
@@ -2378,6 +2422,13 @@ fn eachBind(globalThis: *JSGlobalObject, callframe: *CallFrame) bun.JSError!JSVa
                         tag_to_use = .skipped_because_label;
                     }
                 }
+
+                if (Jest.runner.?.test_options.test_line_filters.count() > 0) {
+                    if (!Jest.runner.?.matchesLineFilter(each_data.line_number, parent)) {
```

Comment:
this should be abstracted into a single method that does the lookup instead of doing both of these checks.